src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,136 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.22.0.dev2 (10/13/2019)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+* Deprecate mypy-version flag and add new version flag (#8463)
+  `Issue #7627 <https://github.com/pantsbuild/pants/issues/7627>`_
+  `PR #8463 <https://github.com/pantsbuild/pants/pull/8463>`_
+
+* Fix tensorflow==1.14 globbing with platform-specific native_lib_names (#8378)
```

Comment:
I think this is a bug fix

Reply:
I put it here because it adds the dict format to the platforms API